src/plugins.ts

```diff
@@ -83,14 +83,16 @@ export async function setupPlugins(server: PluginsServer): Promise<void> {
         }
     }
 
-    if (server.defaultConfigs.length > 0) {
-        server.defaultConfigs.sort((a, b) => a.order - b.order)
-        for (const teamId of Object.keys(server.pluginConfigsPerTeam).map((key: string) => parseInt(key))) {
-            server.pluginConfigsPerTeam.set(teamId, [
+    const sortFunction = (a: PluginConfig, b: PluginConfig) => (a.order || 0) - (b.order || 0)
```

Comment:
Shouldn't `PluginConfig.order` always be a number?

Reply:
It doesn't have `not null` in the database, so 🤷 .

Reply:
So, it will with https://github.com/PostHog/posthog/pull/3122

Reply:
ok, fixed then here as well